.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": [
+        ["env", {
+            "targets": {
+                "browsers": ["last 2 versions", "ie >= 10"]
+            }
+        }]
+    ]
+}
```

Comment:
wow I didn't know this plugin existed, really useful

Reply:
Yeah, brand new stuff!